src/api/todos.ts

```diff
@@ -0,0 +1,27 @@
+import { Todo } from '../types/Todo';
+import { client } from '../utils/fetchClient';
+
+export const getTodos = (userId: number) => {
+  return client.get<Todo[]>(`/todos?userId=${userId}`);
+};
+
+export const createTodo = (todo: Todo) => {
+  return client.post<Todo>('/todos', {
+    userId: todo.userId,
+    title: todo.title,
+    completed: todo.completed,
+  });
+};
+
+export const updateTodos = (todo: Todo) => {
+  return client.patch<Todo[]>(`/todos/${todo.id}`, {
+    userId: todo.userId,
```

Comment:
Actually it's not necessary to pass an id

Reply:
The User Id is what beind passed there, I believe